components/Landing.js

```diff
@@ -2,7 +2,7 @@ import React from "react";
 
 const Landing = () => {
 	return (
-		<div className='flex justify-center items-center bg-acm-white w-full mx-auto mt-32'>
+		<div className='flex justify-center items-center bg-acm-blue w-full mx-auto py-12'>
```

Comment:
Can you make it such that the landing page is also the entire height of the given screen size, so that is all you see once your land on the page. You just need to add `h-[100vh]`

Reply:
mobile is not as ideal, but that is a problem for later!